File: Programming Basics/Programming Basics - C#/Exercises/05. Simple Loops/05. Simple Loops/Odd Even Sum/Odd Even Sum.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Odd_Even_Sum
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.Write("n = ");
            var n = int.Parse(Console.ReadLine());
            var oddSum = 0;
            var evenSum = 0;

            for (int i = 0; i < n; i++)
            {
                var element = int.Parse(Console.ReadLine());
                if (i % 2 ==0)
                {
                    evenSum += element;
                }

                else
                {
                    oddSum += element;
                }
            }

            if (evenSum == oddSum)
            {
                Console.WriteLine("Yes, sum = " + evenSum);
            }

            else
            {
                Console.WriteLine("No, diff = " + Math.Abs(evenSum - oddSum));
            }
        }
    }
}

File: Programming Basics/Programming Basics - JavaScript/Exercises/05. Simple Loops/08. Odd Even Sum/OddEvenSum.js
function checkOddEvenSum(args) {
    let n = Number(args[0]);

    let evenSum = 0;
    let oddSum = 0;

    for (let i = 1; i <= n; i++) {
        let element = Number(args[i]);
        if (i % 2 == 0) {
            evenSum += element;
        }
        else {
            oddSum += element;
        }
    }

    if (evenSum == oddSum) {
        console.log("Yes, sum = " + evenSum);
    }

    else {
        console.log("No, diff = " + Math.abs(evenSum - oddSum));
    }
}

/* Uncomment to test:
checkOddEvenSum(["4", "10", "50", "60", "20"]);
checkOddEvenSum(["4", "3", "5", "1", "-2"]);
checkOddEvenSum(["3", "5", "8", "1"]);
    */